src/fragments/lib/graphqlapi/graphql-from-node.mdx

```diff
@@ -167,67 +170,66 @@ const graphqlQuery = require('./query.js').mutation;
 const apiKey = process.env.API_<YOUR_API_NAME>_GRAPHQLAPIKEYOUTPUT;
 
 exports.handler = async (event) => {
-    const req = new AWS.HttpRequest(appsyncUrl, region);
-
-    const item = {
-        input: {
-            name: "Lambda Item",
-            description: "Item Generated from Lambda"
-        }
-    };
-
-    req.method = "POST";
-    req.path = "/graphql";
-    req.headers.host = endpoint;
-    req.headers["Content-Type"] = "application/json";
-    req.body = JSON.stringify({
-        query: graphqlQuery,
-        operationName: "createTodo",
-        variables: item
-    });
-
-    if (apiKey) {
-        req.headers["x-api-key"] = apiKey;
-    } else {
-        const signer = new AWS.Signers.V4(req, "appsync", true);
-        signer.addAuthorization(AWS.config.credentials, AWS.util.date.getDate());
+  const req = new AWS.HttpRequest(appsyncUrl, region);
+  
+  const item = {
+    input: {
+      name: "Lambda Item",
+      description: "Item Generated from Lambda"
     }
-
-    const data = await new Promise((resolve, reject) => {
-        const httpRequest = https.request({ ...req, host: endpoint }, (result) => {
-            let data = "";
-
-            result.on("data", (chunk) => {
-                data += chunk;
-            });
-
-            result.on("end", () => {
-                resolve(JSON.parse(data.toString()));
-            });
-        });
-
-        httpRequest.write(req.body);
-        httpRequest.end();
+  };
+  
+  req.method = "POST";
+  req.path = "/graphql";
+  req.headers.host = endpoint;
+  req.headers["Content-Type"] = "application/json";
+  req.body = JSON.stringify({
+    query: graphqlQuery,
+    operationName: "createTodo",
+    variables: item
+  });
+  
+  if (apiKey) {
+    req.headers["x-api-key"] = apiKey;
+  } else {
+    const signer = new AWS.Signers.V4(req, "appsync", true);
```

Comment:
Do we use "appsync" or do we have a constant defined anywhere like  AWS_SERVICES.APPSYNC?